diagrams/context.puml

```diff
@@ -73,12 +73,12 @@ services   <-R-> stores     : Query/Transact\n[in terms of events]
 stores       ..> publishers : are followed by
 stores       ..> reactors   : trigger
 
-reactors     --> services   : feed back into
+reactors     --> services   : feed data back into
 reactors     ..> outputs    : generate complex\nnotifications for
 reactors    .U.> caches     : maintain views in
 
 publishers   ..> outputs    : emit complex notifications to
-publishers   ..> outputs    : render some events to
```

Comment:
here, I'm trying to convey that it's being selective in what it opts to render (whereas for complex notifications, it's publishing every one that's been prepared by the "Compose Notifications" umbrella thing (which is another reason why one might not be feeding just `data` above))

Reply:
how about, render filtered events to?

Reply:
![image](https://user-images.githubusercontent.com/206668/78934767-66224000-7aa3-11ea-81b6-e1e1815ab67c.png)